pkgs/love2d/example/assets/love_example_browser/viewcode.lua

```
@@ -0,0 +1,48 @@
+
+	local colors = {keyword = {0,0,1}, string = {0.5,0.5,0.5}, comment = {0,0.5,0},
+		number = {1,0.5,0} , iden = {0,0,0}, default = {0,0,0.5}, line={1,0,0}}
+
+	local contents = love.filesystem.read(filename, 4096)
+	if not contents then
+		contents = "error loading file" .. filename
+		return
+	end
+	
+	maxheight=0							--this should help us in the scrolling thingy
+	for t in contents:gfind("\n") do
```

Luacheck (1.2.0)

[warning] unused loop variable 't'

(W213)